eof/download.py

```diff
@@ -107,22 +91,56 @@ def download_eofs(orbit_dts=None, missions=None, sentinel_file=None, save_dir=".
             query.update(client.query_orbit_for_product(sentinel_file))
         else:
             for mission, dt in zip(missions, orbit_dts):
-                result = client.query_orbit(dt, dt + timedelta(days=1),
-                                            mission, product_type='AUX_POEORB')
+                found_result = False
+                products = client.query_orbit(dt - ScihubGnssClient.T0,
```

Comment:
I added this in my final testing. without the `- ScihubGnssClient.T0`, it was downloading orbits that started on the same day as the sentinel file, and didn't cover the correct validity span